core/services/relay/evm/chain_writer.go

```diff
@@ -6,6 +6,9 @@ import (
 	"math/big"
 	"strings"
 
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
```

Comment:
The linter should be flagging these as unsorted :thinking:

Reply:
Are these not sorted?

Reply:
No, the whole import section should be ordered:
1. std lib
2. third party
3. smartcontractkit
4. this module

It is OK to have multiple groupings within 2 & 3, but packages from the same module should always stay grouped together.

Reply:
Huh, that's strange. When I run `golangci-lint run -v --fix ./core/services/relay/evm` I get no warnings or anything and obviously the CI linting test is catching this. When you lint locally does it fix / warn about this?

Reply:
The tools as configured will only separate std lib, third party, and local module. Keeping things organized beyond that is up to us.

Reply:
Ah ok, I just reordered them manually